docker/benchmarks/run-client.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+#echo on
+set -x
+
+docker run \
+    -d \
+    --log-opt max-size=10m \
+    --log-opt max-file=3 \
+    --name benchmarks-client \
+    --network host \
+    --restart always \
+    benchmarks \
+    /root/.dotnet/dotnet \
+    /benchmarks/src/BenchmarksClient/bin/Debug/netcoreapp2.0/BenchmarksClient.dll
```

Comment:
The Client and Server are fine to run in Debug, they are just web APIs which manage the client and server jobs.  The app we use to measure RPS is always run in Release.